config

```diff
@@ -1,4 +1,4 @@
 [Config]
 location=location_here
 
-metasmoke_host=http://127.0.0.1:3000
+metasmoke_host=dvsbsbsfb
```

Comment:
As [Andy noticed](http://chat.stackexchange.com/transcript/message/29807982#29807982), looks like this shouldn't have been committed.